File: luxoft-jmoncisbays-angular/src/app/models/transaction.ts
import { Company } from './company';

export class Transaction {
  constructor() {};

  public id: number;
  public transactionDate: Date;
  public company: Company;
  public identifier: string;
  public headline: string;
}

export class HttpBodyResponse {
  constructor() {};

  public rowCount: number;
  public pages: number;
  public transactions: Transaction[];
}

File: Luxoft_JMoncisbays_WebAPI/Luxoft_JMoncisbays_WebAPI/Models/Transaction.cs
using System;

namespace Luxoft_JMoncisbays_WebAPI.Models
{
    public class Transaction
    {
        public int Id { get; set; }
        public DateTime TransactionDate { get; set; }
        public int CompanyId { get; set; }
        public string Identifier { get; set; }
        public string Headline { get; set; }

        public Company Company { get; set; }
    }
}
